docs/GDI-SOP_review-checklist.md

```diff
@@ -0,0 +1,80 @@
+pip install --upgrade --no-deps -r requirements.txt
+python3 scripts/sop_linter.py tests/GDI-SOP0000_sop-template_for_linting.md -v 1
+````
+
+Replace the test SOP (``tests/GDI-SOP0000_sop-template_for_linting.md``) with the file path of the SOP you are reviewing.
+
+If there are no linting errors, the output will look like the following:
+````
+{
+  "tests/GDI-SOP0000_sop-template_for_linting.md": {
+    "errors": [],
+    "warnings": []
+  }
+}
+````
+
+For your convenience, the script is automatically executed (via [``lint_sops.yml``](../.github/workflows/lint_sops.yml) workflow) whenever a [Pull Request](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/pulls) (PR) containing SOPs is created. Therefore, if you are reviewing a PR, you can **easily see if the SOPs comply with the expected format** by ensuring that the **latest commit has the green check mark** (✔️) next to it.
+
+![Valid: workflow check in Pull Request](images/GDI-SOP_review-checklist-workflows.png)
+
+If, on the contrary, the latest commit has a red cross (❌), some of the checks did not pass and it requires further inspection.
+
+### 2. **Content Completeness**
+- [ ] **Purpose and Scope**: Ensure the SOP clearly states its purpose and scope, with no ambiguity about the intended outcomes.
+- [ ] **Roles and Responsibilities**: Confirm that the roles are clearly defined and all stakeholders (OC, SDPC, authors, reviewers, etc.) are properly referenced.
+- [ ] **Procedures**: Review all procedures for clarity and completeness. Procedures should include step-by-step instructions that are easy to follow.
+- [ ] **References**: Ensure any external references, tools, or resources are correctly cited and linked if applicable. Make sure that all hyperlinks work as expected.
+
+### 3. **Clarity & Usability**
+- [ ] **Language**: Verify the SOP is written in clear, simple language, avoiding jargon where possible.
+- [ ] **Consistency**: Ensure terms are used consistently across the SOP, particularly those defined in the glossary.
+- [ ] **Examples/Diagrams**: Check for the presence of any necessary examples, diagrams, or clarifying notes to aid understanding.
+
+### 4. **Operational Feasibility**
+- [ ] **Applicability**: Evaluate whether the content of SOP is relevant to the intended operational environment and aligned with GDI objectives.
+- [ ] **Scalability**: Confirm that the SOP is applicable to all GDI nodes, or clearly states any node-specific requirements. This shall go in accordance with its SOP type (i.e., Node-specific or European-level).
+- [ ] **Feasibility**: Ensure that the procedures are feasible for the assigned roles and aligned with the technological capabilities of the nodes.
+
+### 5. **Cross-Referencing and SOP Compliance**
+- [ ] **Referencing Existing SOPs**: Ensure that the SOP does not overlap or conflict with other existing SOPs, and references them if necessary. Find a full list of released SOPs [here](../sops/README.md).
```

Comment:
Since "here" links are considered bad practice, I would suggest replacing the last sentence with for example: `see [full list of released SOPs](../sops/README.md)`